bindings/python/make_torrent.py

```diff
@@ -33,6 +37,7 @@
 		fname = os.path.join(root[len(parent_input)+1:], f)
 		size = os.path.getsize(os.path.join(parent_input, fname))
 		print('%10d kiB  %s' % (size / 1024, fname))
+		# TODO: is deprecated, bind the updated create torrent interface
```

Comment:
are you sure this comment is correct? I was convinced that I fixed all deprecated calls in ``make_torrent.py``